manifests/TeXstudio/TeXstudio/3.0.0alpha5.yaml

```
@@ -0,0 +1,15 @@
+Id: TeXstudio.TeXstudio
+Version: 3.0.0alpha5
+Name: TeXstudio
+Publisher: TeXstudio
+License: GPL-3.0
+LicenseUrl: https://github.com/texstudio-org/texstudio/blob/master/COPYING
+AppMoniker: texstudio
+Tags: tex
+Description: TeXstudio is a fully featured LaTeX editor. Our goal is to make writing LaTeX documents as easy and comfortable as possible.
+Homepage: http://www.texstudio.org/
```

This homepage is also available in HTTPS
```suggestion
Homepage: https://www.texstudio.org/
```